brave/brave-core
# Modifying Rewards Welcome screen for 1 step opt-in process

Fixes: https://github.com/brave/brave-browser/issues/6563

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Run Brave from a fresh profile
2. Navigate to Rewards screen in the welcome process
3. Click `Turn on Rewards`
4. Verify that the text changes to `Turning on...`
5. Verify that you are redirected to `brave://rewards` and that Rewards/Ads is enabled

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
